### PR TITLE
Ensure that the GitHubPaneViewModel is initalized.

### PR DESCRIPTION
**Pull request by grokys**
- Renamed `IGitHubToolWindowManager.ShowHomePane` to `ShowGitHubPane` because the latter is what we call it these days
- Make `IGitHubToolWindowManager.ShowGitHubPane` async to make sure that `GitHubPaneViewModel.InitializeAsync` has finished before passing back a refrence to the `GitHubPaneViewModel`
- Make the methods that call `IGitHubToolWindowManager.ShowGitHubPane` `void async` with logging when they fail

### CODE REVIEW

**Comment by jcansdale** [comment]
@grokys,

- I'm finding the `GitHub` pane spins forever when I open VS with the pane visible:

image

- The `GitHub.ShowCurrentPullRequest` command does seem to work when the `GitHub` pane is visible.  

- If VS is opened with the `GitHub` pane closed, the `GitHub.ShowCurrentPullRequest` command doesn't open the pane.

**Comment by grokys** [comment]
@jcansdale could you take another look at this? It's hopefully working now.

File: src/GitHub.Exports.Reactive/Services/IPullRequestSessionManager.cs [hunk 1]
```diff
@@ -31,6 +31,18 @@ public interface IPullRequestSessionManager : INotifyPropertyChanged
         /// </returns>
         IPullRequestSession CurrentSession { get; }
 
+        /// <summary>
+        /// Ensures that the service is initialized.
+        /// </summary>
+        /// <returns>A task that when completed indicates that the service is initialized.</returns>
+        /// <remarks>
```
Comment (jcansdale) [inline src/GitHub.Exports.Reactive/Services/IPullRequestSessionManager.cs]: Simplying? ;-)

Comment (jcansdale) [reply src/GitHub.Exports.Reactive/Services/IPullRequestSessionManager.cs]: Hmm, is seems I missed the line with `simplying` and it isn't showing up for me as an in-line comment. What's going on? 😕

Comment (meaghanlewis) [reply src/GitHub.Exports.Reactive/Services/IPullRequestSessionManager.cs]: I don't see the above as inline comments either

**Pull request merged**